astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
```

Comment:
Is it worth splitting out the formula in a standalone function that doesn't depend on `astroplan` objects?
Then call it from this method.

This is done in Astropy e.g. for `separation` and a few other places ... makes re-use easier.
Although if there's no second caller right now, it's not really needed and maybe isn't a good idea.
Just a thought.